vlasiator.cpp

```diff
@@ -1103,18 +1128,17 @@ int simulate(int argn,char* args[]) {
          // moved.
          SBC::ionosphereGrid.updateIonosphereCommunicator(mpiGrid, technicalGrid);
       }
-      
+
       //get local cells
       const vector<CellID>& cells = getLocalCells();
 
       //compute how many spatial cells we solve for this step
       computedCells=0;
       for(size_t i=0; i<cells.size(); i++) {
          for (uint popID=0; popID<getObjectWrapper().particleSpecies.size(); ++popID)
-            computedCells += mpiGrid[cells[i]]->get_number_of_velocity_blocks(popID)*WID3;
+            computedCells += (uint64_t)mpiGrid[cells[i]]->get_number_of_velocity_blocks(popID)*WID3;
       }
```

Comment:
Any clue if we should thread this loop at least? And put the *=WID3 outside of it? Or find a suitable other loop to piggyback and tuck that in there?